File: Game/src/main/java/net/winternetwork/bedwars/game/settings/GameSettings.java
package net.winternetwork.bedwars.game.settings;

import net.winternetwork.bedwars.api.game.stage.Stage;
import net.winternetwork.bedwars.game.module.stage.model.GameStartedStage;
import net.winternetwork.bedwars.game.module.stage.model.ToStartStage;
import net.winternetwork.bedwars.game.module.stage.model.WaitingPlayerStage;

public class GameSettings {

    public static final int PLAYERS_TO_START = 4;
    public static final int MAX_PLAYERS = 4;

    public static final Stage[] STAGE_ARRAY = {
            new WaitingPlayerStage(),
            new ToStartStage(),
            new GameStartedStage()
    };
}

File: Game/src/main/kotlin/net/winternetwork/bedwars/game/settings/GameSettings.kt
package net.winternetwork.bedwars.game.settings

import com.google.common.collect.ImmutableList
import net.winternetwork.bedwars.game.modules.stage.model.GameStartedStage
import net.winternetwork.bedwars.game.modules.stage.model.ToStartStage
import net.winternetwork.bedwars.game.modules.stage.model.WaitingPlayerStage

object GameSettings {

    const val PLAYERS_TO_START = 4
    const val MAX_PLAYERS = 4

    val STAGE_ARRAY = ImmutableList.of(
            WaitingPlayerStage(),
            ToStartStage(),
            GameStartedStage()
    )
}